README.md

```diff
@@ -850,6 +931,12 @@ TODO-B: - translation present
 TODO-A: Problems:
 TODO-A: - TBD
 TODO-B: Problems: None
+
+[Re-Review*]
```

Comment:
Should there be a requirement to see if upstream code is still maintained? e.g., that a git repo is not in archive mode

Perhaps this isn't `Re-Review` specific.